sql/auto_install.sql

```diff
@@ -44,3 +44,57 @@ CREATE TABLE IF NOT EXISTS `civicrm_case_category_features` (
   PRIMARY KEY (`id`),
   UNIQUE INDEX `unique_category_feature` (category_id, feature_id)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
```

Comment:
It is better for any custom defined entity to have a table name that starts with the extension name, to prevent confusing such tables with core tables for anyone who is looking at the database, so these should be:

civicase_sales_order
civicase_sales_order_line

I can see there are other old tables that use `civicrm_case_` , but I don't think we need to follow that and that any new table in Civicase should be clear that it is coming from Civicase.

Reply:
great 👍, it's fine we follow this approach from now.